### PR TITLE
Alternative intro

### PR DESCRIPTION
**Pull request by samuelcolvin**
Alternative to #260, cc @sydney-runkle @summerscope @dmontagu?

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -2,11 +2,13 @@
 
 --8<-- "docs/.partials/index-header.html"
 
-When I first found FastAPI, I got it immediately. I was excited to find something so innovative and ergonomic built on Pydantic.
+**PydanticAI is a Python Agent Framework designed to make it less painful to build production grade applications with Generative AI.**
 
-Virtually every Agent Framework and LLM library in Python uses Pydantic, but when we began to use LLMs in [Pydantic Logfire](https://pydantic.dev/logfire), I couldn't find anything that gave me the same feeling.
+FastAPI revolutionized web development by offering an innovative and ergonomic design, built on the foundation of [Pydantic](https://docs.pydantic.dev).
```
Comment (sydney-runkle) [inline docs/index.md:8]:
```suggestion
[FastAPI](https://fastapi.tiangolo.com/) revolutionized web development by offering an innovative and ergonomic design, built on the foundation of [Pydantic](https://docs.pydantic.dev).
```

File: docs/index.md [hunk 2]
```diff
@@ -2,11 +2,13 @@
...
 
-PydanticAI is a Python Agent Framework designed to make it less painful to build production grade applications with Generative AI.
+Similarly, virtually every Agent Framework and LLM library in Python uses Pydantic, yet when we began to use LLMs in [Pydantic Logfire](https://pydantic.dev/logfire), we couldn't find anything that gave me the same feeling.
```
Comment (Bradley-Butcher) [inline docs/index.md]:
```suggestion
Similarly, virtually every Agent Framework and LLM library in Python uses Pydantic, yet when we began to use LLMs in [Pydantic Logfire](https://pydantic.dev/logfire), we couldn't find anything that gave us the same feeling.
```

File: docs/index.md [hunk 3]
```diff
@@ -2,11 +2,13 @@
 
 --8<-- "docs/.partials/index-header.html"
 
-When I first found FastAPI, I got it immediately. I was excited to find something so innovative and ergonomic built on Pydantic.
+**PydanticAI is a Python Agent Framework designed to make it less painful to build production grade applications with Generative AI.**
```
Comment (summerscope) [inline docs/index.md]:
```suggestion
**PydanticAI is a Python agent framework designed to make it less painful to build production-ready applications with Generative AI.**
```

Not sure why we're using caps on `agent` and `framework`?

File: docs/index.md [hunk 4]
```diff
@@ -2,11 +2,13 @@
...
 
-Virtually every Agent Framework and LLM library in Python uses Pydantic, but when we began to use LLMs in [Pydantic Logfire](https://pydantic.dev/logfire), I couldn't find anything that gave me the same feeling.
+FastAPI revolutionized web development by offering an innovative and ergonomic design, built on the foundation of [Pydantic](https://docs.pydantic.dev).
 
-PydanticAI is a Python Agent Framework designed to make it less painful to build production grade applications with Generative AI.
+Similarly, virtually every Agent Framework and LLM library in Python uses Pydantic, yet when we began to use LLMs in [Pydantic Logfire](https://pydantic.dev/logfire), we couldn't find anything that gave us the same feeling.
+
+We built PydanticAI with one simple aim: to bring the same feeling you get building with FastAPI to GenAI app development.
```
Comment (summerscope) [inline docs/index.md]:
```suggestion
We built PydanticAI with one simple aim: to bring that FastAPI feeling to GenAI app development.
```

Just trying to tighten this up, and make it punchier. IDK if the shorthand 'FastAPI feeling' is too much focus on them?

**Review by summerscope — APPROVED** [review]
Pushed a styling change, suggested some text edits - commit or ignore as you like :) 👍

File: docs/index.md (hunk 1 above)
Comment (samuelcolvin) [reply docs/index.md:8]: I don't think the first link in the page should be to an external project.

File: docs/index.md (hunk 3 above)
Comment (samuelcolvin) [reply docs/index.md]: changed to `agent framework`, kept `production grade`.

**Pull request merged**